sencha-workspace/SlateTasksManager/sass/src/view/TasksManager.scss

```diff
@@ -1,9 +1,23 @@
+.content {
+  position: relative;
+  height: calc(100vh - 60px);
+  width: 100vw;
+}
+
+#slateapp-viewport {
+  height: 100%;
+  position: absolute;
+  top: 0px;
+  bottom: 0px;
+  width: 100vw;
+}
+
```

Comment:
in stuff within `app/sass/{src,var}`, each file corresponds to a component and it's best to avoid including any CSS anywhere in this tree that wouldn't be scoped to the component the file is for. Notice the version before this only had stuff within `.slate-tasks-manager`

`app/sass/etc/all.scss` is the place for random global stuff

In any case, I found this calculation to be off a bit and leaving gaps in the window, so I've replicated how SlateAdmin over in the `slate` repository is set up and did some other clean up: https://github.com/SlateFoundation/slate-cbl/pull/684/files/b6b42921ddcf39a45ec8d20e35fa007c7091f860..e7544db3d673dbfa063bac38ad6d8d3c65242a21